src/components/typography/typography.stories.js

```diff
@@ -0,0 +1,9 @@
+import "./typography.css";
+import typography from "./typography.html";
+
+export default {
+  title: "Components/Typography",
+  parameters: { layout: "centered" },
+};
+
+export const dog = () => typography;
```

Comment:
🐶 I think this should be named differently

Reply:
thx, done!